deploy/packages/bbp-packages.yaml

```diff
@@ -74,7 +74,7 @@ packages:
     specs:
       - functionalizer@3.12.2
       - parquet-converters@0.5.2
-      - synapsetool@0.5.6
+      - synapsetool
```

Comment:
I think this creates the `synapsetool` modules… better to pin the latest version.

Reply:
I thought this was asking to install last version. How does it work then?